### PR TITLE
Use a consistent style everywhere.

### PR DESCRIPTION
**Pull request by emilio**
r? @fitzgen

### CODE REVIEW

**Comment by emilio** [comment]
I'm still unsure about the "run it on CI" thing. It's nice but it seems it can be pretty annoying for new contributors. I'd say it's worth trying, but if we see too many people getting stuck with it, we can back it out.

**Comment by fitzgen** [comment]
AWESOME!!

I was going to bring this up soon, and I actually accidentally `rustfmt`ed some files locally because of key chord habits in my editor :P

Taking a look now...

**Comment by fitzgen** [comment]
> I'm still unsure about the "run it on CI" thing. It's nice but it seems it can be pretty annoying for new contributors. I'd say it's worth trying, but if we see too many people getting stuck with it, we can back it out.

The other thing is, if we have different versions of `rustfmt` installed, they can end up formatting the code differently. I've run into this with `gimli`, where we decided that it was not worth it to put in CI.

Soft -1 about running in CI.

Ideally, bors or highfive or whoever would automatically run rustfmt after every merge, and if there was any change, then automatically merge a `rustfmt` commit. Best of both worlds: we don't bounce new contributors and we always end up with a styled master.

**Comment by emilio** [comment]
@fitzgen sounds good, I'll drop the CI commit. Meanwhile we can run it manually I guess.

File: .travis.yml [hunk 1]
```diff
@@ -39,6 +40,8 @@ before_script:
   - echo $LIBCLANG_PATH
 
 script:
+  - export PATH=$PATH:~/.cargo/bin
+  - cargo fmt -- --write-mode=diff
```
Comment (fitzgen) [inline .travis.yml]: Soft -1 if this is going to cause travis CI to fail if there is any difference in the formatting. As mentioned before, we will hit this issue with different `rustfmt` versions.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ yourself.
 * [Code of Conduct](#coc)
 * [Filing an Issue](#issue)
 * [Building](#building)
+* [Automatic Code Formatting](#formatting)
```
Comment (fitzgen) [inline CONTRIBUTING.md]: Maybe put this after testing? I feel like learning about testing is more important for newcomers.

Comment (emilio) [inline CONTRIBUTING.md]: Yup, agreed! Done :)

**Comment by emilio** [comment]
For some reason bors didn't listen to me initially. Meh.

@bors-servo r=fitzgen

**Comment by emilio** [comment]
Reopened as https://github.com/servo/rust-bindgen/pull/183, given homu was ignoring this one.

**Pull request closed without merging**